src/boot/viconfig.c

```diff
@@ -2,16 +2,20 @@
 #include "vt.h"
 
 // this should probably go elsewhere but right now viconfig.o is the only object between idle and z_std_dma
-OSPiHandle* gCartHandle = 0;
+OSPiHandle* gCartHandle = NULL;
 
-void ViConfig_UpdateVi(u32 mode) {
-    if (mode != 0) {
+void ViConfig_UpdateVi(u32 black) {
+    if (black) {
+        // Black the screen on next call to ViConfig_UpdateBlack
         osSyncPrintf(VT_COL(YELLOW, BLACK) "osViSetYScale1(%f);\n" VT_RST, 1.0f);
```

Comment:
I put it here to highlight that it will update the VI configuration in a different way, depending on if the screen will be blacked out. I've tried adding to the comment to hopefully clarify this.